torch_xla/__init__.py

```diff
@@ -78,6 +78,10 @@ def _summarize_fn_tracker():
 
 
 def _aws_ec2_inf_trn_init():
+  import importlib.util
+  if importlib.util.find_spec("torch_neuronx") is not None:
+    os.environ["DISABLE_NUMERIC_CC_TOKEN"] = "1"
```

Comment:
can we move it to the else part of the below try except else

Reply:
Let me test that out first.

Reply:
Tested that setting within neuron.initialize_env gets picked up by GetInput and GetNewToken. Updated. Thanks!